internal/domains/auth.go

```diff
@@ -0,0 +1,56 @@
+package domains
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-shiori/shiori/internal/database"
+	"github.com/go-shiori/shiori/internal/model"
+	"github.com/golang-jwt/jwt/v4"
+	"go.uber.org/zap"
+	"golang.org/x/crypto/bcrypt"
+)
+
+type AuthDomain struct {
```

Comment:
It's a good idea to use domain to seperate common login between cli and web api. Is the name `domain` a usual one? I usually use `service`.

Reply:
I tend to use `domain` when `service` can be missinterpreted or used for other things, so I just went with it. Right now I think is irrelevant and I could change it before going forward with this.

Reply:
I agree. The name `service` is used everywhere now.